cloud/aws/modules/ecs_fargate_service/variables.tf

```diff
@@ -149,3 +149,12 @@ variable "extra_inbound_rule_cidr" {
   type        = string
   default     = null
 }
+
+variable "healthcheck" {
```

Comment:
**suggestion:** Thoughts on calling this lb_healthcheck or something to make it a bit more specific?